install.sh

```diff
@@ -29,14 +29,12 @@ if [ "$(id -u)" != "0" ]; then
     [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
 fi
 
-# Split path to this install script into an array.
-IFS='/' SRC_PATH=($0)
-# Remove this file name.
-unset SRC_PATH[-1]
+#Create opt folder under root
+mkdir -p /opt
+#Get current shell script directory name
+SRC_PATH=$(cd "$(dirname "$0")"; pwd)
 # Grab the directory name.
-APP_DIR="${SRC_PATH[-1]}"
-# Joint path back together.
-SRC_PATH="${SRC_PATH[*]}"
+APP_DIR=${PWD##*/}
```

Comment:
This looks like it will set the APP_DIR to the directory the script was executed from (rather than the directory with the install.sh script). Those are the same if the script is executed from the same directory as the script.

A fix may be to move the 
```cd "$(dirname "$0")"```
out out of line 35 (maybe put it on its own line above line 35).
That will change the PWD that is used in line 37.